scripts/rich-core-dumper

```diff
@@ -64,7 +64,8 @@ _obtain_configuration()
   # default fallback values, overruled by configuration file if one exists
 
   system_settings=/usr/share/crash-reporter-settings
-  user_settings=/home/nemo/.config/crash-reporter-settings
+  system_user=$(getent group sailfish-system | cut -d: -f4 | tr , '\n' | grep -vE '^sailfish-')
```

Comment:
The other option would be to take intersection from sailfish-system and users groups' users.

Reply:
Even head -n1 doesn't make sure it's current user if for some reason there are several sailfish-system users. Works but have to be careful that there can be only one.

Reply:
Indeed, but also "current user" doesn't make sense here because we really only want the device owner to be the only one that can change or override these settings.

Of course it's possible to read settings from multiple users' homes but then what is the priority? Whose settings would be used if there are multiple? Anyway the plan is to have just one device owner, so there needs to be some kind of interface to get that value.

Reply:
Yep, this seems to be the way to do this at the moment. Didn't work for me but I have my users created manually and just copies of nemo.  LGTM.